config/packages/_sylius.yaml

```diff
@@ -16,3 +16,7 @@ sylius_api:
 sylius_shop:
     product_grid:
         include_all_descendants: true
+
+sylius_resource:
+    settings:
+        state_machine_component: symfony
```

Comment:
To remove of course, User has to choose itself

Reply:
Wouldn't it be ok to make it the new default value tho?
The upgrade guide needs to get a new part about this BC but new projects should use symfony/workflow, shouldn't they?

Reply:
Wouldn't it be ok to make it the new default value tho?
The upgrade guide needs to get a new part about this BC but new projects should use symfony/workflow, shouldn't they?

Reply:
Hum, by default, the state machine component is set up to winzou when both are available.
This configuration could stay like this cause it does not impact end-user application.

But still, it should stay with Winzou for 1.x release at least for 1.13.

Reply:
But in Sylius standard we'll be able to set this value to use Symfony Workflow

Reply:
How to ensure both workflow system are maintained ? We can't run all test twice

Reply:
We can "include" a test with Workflow it's easy to do.